ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -0,0 +1,600 @@
+import com.spectralogic.ds3client.models.*;
+import com.spectralogic.ds3client.models.Job;
+import com.spectralogic.ds3client.models.bulk.Ds3Object;
+import com.spectralogic.ds3client.serializer.XmlProcessingException;
+import com.spectralogic.ds3client.utils.ResourceUtils;
+import static org.hamcrest.Matchers.*;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.*;
+import java.security.SignatureException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import static com.spectralogic.ds3client.helpers.Ds3ClientHelpers.*;
+import static com.spectralogic.ds3client.integration.Util.*;
+import static org.junit.Assert.*;
+
+public class PutJobManagement_Test {
+
+    private static final Ds3Client client = Util.fromEnv();
+    private static final Ds3ClientHelpers HELPERS = Ds3ClientHelpers.wrap(client);
+    private static final String BUCKET_NAME = "Put_Job_Management_Test";
+
+    private static final String TEST_ENV_NAME = "PutJobManagement_Test";
+    private static TempStorageIds envStorageIds;
+
+    @BeforeClass
+    public static void startup() throws IOException, SignatureException {
+        //client = Util.fromEnv();
+        final UUID dataPolicyId = TempStorageUtil.setupDataPolicy(TEST_ENV_NAME, true, ChecksumType.Type.MD5, client);
+        envStorageIds = TempStorageUtil.setup(TEST_ENV_NAME, dataPolicyId, client);
+    }
+
+    @Before
+    public void setupBucket() throws IOException, SignatureException {
+        HELPERS.ensureBucketExists(BUCKET_NAME);
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException, SignatureException {
+        TempStorageUtil.teardown(TEST_ENV_NAME, envStorageIds, client);
+        client.close();
+    }
+
+    private void checkTimeOut(final long startTime, final int testTimeOutSeconds){
+        assertThat((System.nanoTime() - startTime)/1000000000, lessThan((long) testTimeOutSeconds));
+    }
+
+    @Test
+    public void nakedS3Put() throws IOException, SignatureException, XmlProcessingException, URISyntaxException {
+        try {
+            final Path beowulfPath = ResourceUtils.loadFileResource(RESOURCE_BASE_NAME + "beowulf.txt");
+            final SeekableByteChannel beowulfChannel = new ResourceObjectPutter(RESOURCE_BASE_NAME).buildChannel("beowulf.txt");
+            final PutObjectResponse job = client.putObject(new PutObjectRequest(BUCKET_NAME, "beowulf.txt",
+                    beowulfChannel, Files.size(beowulfPath)));
+            assertThat(job.getStatusCode(), is(200));
+        } finally {
+            deleteAllContents(client, BUCKET_NAME);
+        }
+    }
+
+    @Test
+    public void getActiveJobs() throws IOException, SignatureException, XmlProcessingException, URISyntaxException {
```

Comment:
From the BP's standpoint, the response contents should be the same.  However, this should take a different code path in the SDK, right?